### PR TITLE
chrome-cli: add support for chromium

### PR DESCRIPTION
**Pull request by dardo82**
It's my first (serious) pull request to this project, i hope i haven't done anything wrong (this time).

### CODE REVIEW

**Comment by dardo82** [comment]
See https://github.com/Homebrew/homebrew/issues/42349 for more info.

File: Library/Formula/chrome-cli.rb [hunk 1]
```diff
@@ -22,11 +22,18 @@ def install
...
     # Canary builds; see:
-    # https://github.com/prasmussen/chrome-cli/issues/15#issuecomment-35202217
+    # https://github.com/prasmussen/chrome-cli/issues/15
     rm_rf "build"
     inreplace "chrome-cli/App.m", "com.google.Chrome", "com.google.Chrome.canary"
     xcodebuild "SDKROOT=", "SYMROOT=build"
     bin.install "build/Release/chrome-cli" => "chrome-canary-cli"
+
+    # Chromium builds; see:
+    # https://github.com/prasmussen/chrome-cli/issues/42349
```
Comment (DomT4) [inline Library/Formula/chrome-cli.rb]: I get a 404 on this link?

**Comment by DomT4** [comment]
The seemingly dead link aside, this looks perfect :+1:.

**Comment by dardo82** [comment]
Sorry for the dumb mistake... :-P

File: Library/Formula/chrome-cli.rb (hunk 1 above)
Comment (dardo82) [inline Library/Formula/chrome-cli.rb]: My bad for the wrong link... :-/

**Comment by DomT4** [comment]
No need for apologies, we all make little mistakes!

Merged in https://github.com/Homebrew/homebrew/commit/68b786041126931589f9f646513da544fa2bc248. Thank you for your contribution to Homebrew, we appreciate it! :smiley_cat:.

Just a nit, but ideal commit message style here would have been something like `chrome-cli: add support for chromium`, Thanks :)

**Comment by dardo82** [comment]
We also make big mistakes? (See issue https://github.com/Homebrew/homebrew/issues/42848 for info.)

**Pull request closed without merging**